delivery/overview.mdx

```diff
@@ -105,7 +105,7 @@ When creating an `upsert` sink, note whether or not you need to specify the prim
 <Note>
 **PUBLIC PREVIEW**
 
-Sink data in parquet encode is in the public preview stage, meaning it's nearing the final product but is not yet fully stable. If you encounter any issues or have feedback, please contact us through our [Slack channel](https://www.risingwave.com/slack). Your input is valuable in helping us improve the feature. For more information, see our [Public preview feature list](/changelog/product-lifecycle#features-in-the-public-preview-stage).
+Sink data in Parquet encode is in the public preview stage, meaning it's nearing the final product but is not yet fully stable. If you encounter any issues or have feedback, please contact us through our [Slack channel](https://www.risingwave.com/slack). Your input is valuable in helping us improve the feature. For more information, see our [Public preview feature list](/changelog/product-lifecycle#features-in-the-public-preview-stage).
```

Comment:
@WanYixian Let's always refer to Parquet as a format. "Sinking data in Parquet format..."